pkg/vm/builtins/keccak_math.go

```diff
@@ -0,0 +1,411 @@
+package builtins
+
+import "math/bits"
+
+// Slighly modified from: https://github.com/golang/crypto/blob/master/sha3/keccakf.go
```

Comment:
What modifications we required?

Reply:
As I remember I had to change it to work on values instead of pointers during testing. Looking at it now it works the just fine with the original code so I'll change it back to that for easier compatibility if go's keccakf1600 ever gets updated.